### ISSUE TITLE
Add option to entirely disable the workspace name / activities button

### ISSUE THREAD

**Issue by raiguard** [paperwm/PaperWM#387]
PaperWM usually has a workspace name in the top bar, along with workspace names baked into each workspace's background. I have a different wallpaper per workspace so I don't need any of these identifiers. I found a way to disable the workspace names, but unfortunately that switches the workspace name to "activites" akin to the stock gnome button.

However, this button is added by PaperWM, so normal extensions to remove this button don't work. I would love a way to completely disable this button, because I never use it and it's just sitting there being unused.

![image](https://user-images.githubusercontent.com/3515394/126918496-1d0eb5c3-8de7-4bd2-9d83-bcc391b254dd.png)

**Comment by ludvigng** [comment]
Same. It also shows up on the wallpaper if you shrink or hide the panel, and will sometimes mess with the placement of other extensions (such as arc menu) if you connect/disconnect another monitor.

**Comment by jtaala** [comment]
Agree, for later PaperWM features (like #476, #482, #499) users can disable those components (or icons in topbar etc.).  This should also be disabiliable (is that word?).
